src/middleware/github-webhook-middleware.test.ts

```diff
@@ -104,5 +126,10 @@ describe("Probot event middleware", () => {
 		context.log.info("test");
 		expect(loggedStuff).toContain("foo");
 		expect(loggedStuff).toContain("123");
+
+		await GithubWebhookMiddleware(jest.fn())(undefined, context);
+		context.log.info("test");
```

Comment:
Is this meant to be here?

Reply:
Yes, testing child log data preserves or not.